packages/core-data/src/hooks/use-entity-record.ts

```diff
@@ -44,7 +45,6 @@ export interface Options {
  * @param  name                   Name of the entity, e.g. `plugin` or a `post`. See rootEntitiesConfig in ../entities.ts for a list of available names.
  * @param  recordId               ID of the requested entity record.
  * @param  options                Optional hook options.
- * @param  [options.enabled=true] Whether to run the query or short-circuit and return null. Defaults to true.
```

Comment:
Docgen does not like this notation in TypeScript. I removed this line since it's documented on the relevant data type.

Reply:
Yes, that's a good example where in TS world you don't need to document the same thing twice because you can infer it from the `Options` definition.